api-docs/rst/dev-guide/api-operations/methods/post-resume-v1.0-tenantid-groups.groupid.rst

```diff
@@ -1,16 +1,20 @@
 
-.. _post-resume-group-policy-execution-v1.0-tenantid-groups-groupid-resume:
 
-Resume group policy execution
+.. _post-resume-v1.0-tenantid-groups-groupid:
+
+Resume group
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. code::
 
-    POST /v1.0/{tenantId}/groups/{groupId}/resume
+    POST /v1.0/{tenantId}/groups/groupId/resume
 
-This operation resumes the execution of all scaling policies for a specified scaling group.
+This operation resumes a scaling group. When a scaling group is resumed, policy
+executions and convergence triggering is allowed and group state returns 
+``"paused": false``. Use :ref:`pause <pause-group>` API operation to pause a group.
 
```

Comment:
This operation resumes a scaling group. When a group is resumed, policy executions and convergence operations are allowed. The group state returns to ``paused=false``.   You can pause a group by submitting a pause <post-pause-v1.0-tenantid-groups-groupid> request.